### PR TITLE
Add tests "how to" Readme

### PR DESCRIPTION
**Pull request by loredanacirstea**

### CODE REVIEW

File: raiden_contracts/tests/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        - raiden core contracts files
+    - data - compiled contracts data & deployment information
+    - data_pre_limits - compiled contracts data & deployment information for an older version with no limits
+    - deploy - deployment & verification scripts
+    - tests
+        - fixtures - fixtures used by all tests
+        - property - property tests for core contracts
+        - unit - unit tests for internal functions in core contracts
+        - utils - specific utilities for tests, closely related to the contracts logic
+        - main test files for both core & service contracts
```
Comment (czepluch) [inline raiden_contracts/tests/README.md]: Is there a `-` missing after `main`?

File: raiden_contracts/tests/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+- when implementing a new feature or changing code, make sure your PR also contains tests that cover the changes.
+- always think about edge cases, go over each Solidity source code line that was changed in your PR and make sure a test covers it if needed.
+- don’t only test the happy case. If you don't have `with pytest.raises():` in your tests, something is wrong
+
+
+### How to Start Testing a Contract
+
+- create a `test_contract_name.py` test file for:
+    - testing the constructor & all public variables for that contract (including the version) - see [TokenNetwork tests](/raiden_contracts/tests/test_token_network.py)
+- if the contract is big - create multiple `test_contract/[contract]_contract_function_name.py` test file for testing each contract function separately. E.g. `test_channel_open.py` (we don't have the contract name here, but we have `channel`, which is specific enough)
```
Comment (czepluch) [inline raiden_contracts/tests/README.md]: test file -> test files

**Comment by loredanacirstea** [comment]
Thank you @czepluch, fixed.
Added some more links to make it clear where to find needed fixtures, added a reference to this file in the main README.

File: raiden_contracts/tests/README.md (hunk 1 above)
Comment (czepluch) [reply raiden_contracts/tests/README.md]: Can you please comment on this or fix it?

Comment (loredanacirstea) [reply raiden_contracts/tests/README.md]: no need for a `-`

Comment (loredanacirstea) [reply raiden_contracts/tests/README.md]: As you see, there is no `main` folder.

Comment (czepluch) [reply raiden_contracts/tests/README.md]: Ah, not I get it. Thanks.
